web/src/views/DandisetLandingView/ContactDialog.vue

```diff
@@ -0,0 +1,130 @@
+          mdi-chevron-down
+        </v-icon>
+      </v-btn>
+    </template>
+    <v-card
+    >
+      <v-card-title class="pb-0" style="min-width: fit-content;">
+        Select an e-mail Recipient!
+      </v-card-title>
+      <v-list>
+        <v-tooltip
+          :disabled="!owners ? false : loggedIn()"
+          open-on-hover
+          right
+        >
+          <template #activator="{ on }">
+            <div
+            v-on="on"
+            >
+              <v-list-item
+                :disabled="!loggedIn() || !owners"
+                :selectable="!loggedIn() || !owners"
+                :href="makeTemplate(owners)"
+              >
+                <v-icon
+                  color="primary"
+                  left
+                  small
+                >
+                  mdi-card-account-mail
+                </v-icon>
+                Dandiset Owners
+              </v-list-item>
+            </div>
+          </template>
+          <span v-if="!loggedIn()"> You must me logged in to contact the owner </span>
+          <span v-if="!owners"> No owner e-mail available </span>
+        </v-tooltip>
+      <v-divider />
+        <v-tooltip
+            :disabled="contacts === undefined || contacts?.length > 0 "
+            open-on-hover
+            right
+          >
+
+          <template #activator="{ on }">
+            <div v-on="on">
+              <v-list-item
+                :disabled=" contacts?.length === 0 || !contacts"
+                :selectable="!contacts"
+                :href="makeTemplate(contacts)"
+              >
+                <v-icon
+                  color="primary"
+                  left
+                  small
+                >
+                  mdi-card-account-mail
+                </v-icon>
+                Contact Person
+              </v-list-item>
+            </div>
+            </template>
+          <span> No contact e-mail available </span>
+        </v-tooltip>
+      </v-list>
+    </v-card>
+  </v-menu>
+</template>
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useDandisetStore } from '@/stores/dandiset';
+import { loggedIn } from '@/rest';
+import type { User, Person, Organization, Email} from '@/types';
+
+const store = useDandisetStore();
+const owners = computed(() => store.owners?.map((owner: User) => owner.email));
+const contacts = computed(() =>
+  store.dandiset?.metadata?.contributor?.filter(
+    (contact: Person | Organization) =>
+      contact.roleName?.includes("dcite:ContactPerson"))
+    .map((contact: Person | Organization) =>
+      contact.email as Email
+    )
+);
+const currentDandiset = computed(() => store.dandiset);
+
+const makeTemplate = (contacts: string[] | undefined) => {
+  if (contacts === undefined) {
+    throw new Error('Contact is undefined.');
+  }
+  if (currentDandiset.value){
+    const subject = encodeURIComponent(`Regarding Dandiset ${currentDandiset.value.name} ${currentDandiset.value.metadata?.id}`);
```

Comment:
The subject line ends up rendering as `Regarding Dandiset NeuroConv SDK Testing DANDI:200560/draft` which feels jumbled up. A better template might be:

```suggestion
    const subject = encodeURIComponent(`Regarding Dandiset ${currentDandiset.value.dandiset.identifier} ("${currentDandiset.value.name}")`);
```

This names the Dandiset by its identifier (i.e. its six-digit, zero-padded ID) which is normally how we refer to Dandisets, and puts its name in quotes within parentheses. So more like `Regarding Dandiset 200560 ("NeuroConv SDK Testing")`.